datastream-kafka/src/test/java/com/linkedin/datastream/kafka/TestKafkaTransportProvider.java

```diff
@@ -220,9 +220,9 @@ public void testSendMultipleEventsInSingleDatastreamProducerRecord() throws Exce
 
     DatastreamTask task = new DatastreamTaskImpl(Collections.singletonList(ds));
     TransportProvider transportProvider = provider.assignTransportProvider(task);
-    provider.createTopic(destinationUri, 1, new Properties());
+    provider.createTopic(destinationUri, 1, new Properties(), ds);
 
-    KafkaTestUtils.waitForTopicCreation(_zkUtils, topicName, _kafkaCluster.getBrokers());
+    //KafkaTestUtils.waitForTopicCreation(_adminClient, topicName, _kafkaCluster.getBrokers());
```

Comment:
Will remove this in following update to this patch. Same for another instance below.